idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/AddModifierFix.kt

```diff
@@ -113,6 +113,7 @@ open class AddModifierFix(
             AddModifierFix.createFactory(KtTokens.OPEN_KEYWORD, KtClassOrObject::class.java)
         val addFinalToProperty: QuickFixesPsiBasedFactory<PsiElement> =
             AddModifierFix.createFactory(KtTokens.FINAL_KEYWORD, KtProperty::class.java)
+        val addInnerModifier = createFactory(KtTokens.INNER_KEYWORD)
```

Comment:
Nit: we should probably unify on one style here. Either put in types on the other lines or simplify the other lines as well (I prefer the latter)?